core/tee/tee_svc_cryp.c

```diff
@@ -1844,6 +1844,15 @@ static TEE_Result tee_svc_cryp_check_key_type(const struct tee_obj *o,
 	case TEE_MAIN_ALGO_DH:
 		req_key_type = TEE_TYPE_DH_KEYPAIR;
 		break;
+	case TEE_MAIN_ALGO_ECDSA:
+		if (mode == TEE_MODE_VERIFY)
+			req_key_type = TEE_TYPE_ECDSA_PUBLIC_KEY;
+		else
+			req_key_type = TEE_TYPE_ECDSA_KEYPAIR;
+		break;
+	case TEE_MAIN_ALGO_ECDH:
+		req_key_type = TEE_TYPE_ECDH_KEYPAIR;
+		break;
```

Comment:
Should  TEE_MAIN_ALGO_ECDH/TEE_MAIN_ALGO_DSA cases be updated (same as TEE_MAIN_ALGO_ECDSA) ?